File: lib/boilerplate-main-code/Array_Input.py
# Main code
testCases = int(input(""));
assert(testCases > 0);

while(testCases >0):

    size = int(input(""));
    arr = [];

    arr = input("").split();
    
    for x in range(size):
        arr[x] = int(arr[x]);
    
    freq = frequencyCount(arr);

    for x in arr:
        print(f"{x} : {freq[x-1]}");

    testCases -= 1;

File: lib/boilerplate-main-code/Array_Input.java
package lib.Main_Boilerplate;

import java.io.*;

public class Array_Input {
    
    public static void main(String[] args) {

        // BufferReader Initialized
        try (BufferedReader br = new BufferedReader(new InputStreamReader(System.in));) {
            
            // Reading number of test cases
            int t = Integer.parseInt(br.readLine());


            while (t-- > 0) {

                // Reading raw string
                String[] tokens = br.readLine().split(" ");
                
                // Creating an array of input size
                int arr[] = new int[tokens.length];

                // Parsing array elements from string
                for (int i = 0; i < tokens.length; i++) {
                    arr[i] = Integer.parseInt(tokens[i]);

                }

                // Call the function
                
            }
        } catch (IOException e) {
            e.printStackTrace();
        }

    }
}
